app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -194,3 +194,9 @@ $user-avatar-mini-border-radius:                50px !default
 $user-avatar-mini-width:                        20px !default
 
 $select-element-padding:                        3px, 24px, 3px, 3px
+
+$homescreen-content-bg-color:                   white
+$homescreen-block-bg-color:                     white
+$homescreen-footer-bg-color:                    #F8F8F8
```

Comment:
There are quite a few occurences of `#F8F8F8` in the global variables.

Not sure if it is a good idea to either assign one of the already existing ones here or to introduce a common color that most of those occurences use.